guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -451,8 +451,11 @@ Use this option if you have one of the following errors:
 . In the {Project} web UI, navigate to *Content* > *Products*.
 . Select the product containing the corrupted repository.
 . Select the name of a repository you want to synchronize.
-. From the *Select Action* menu, select *Advanced Sync*.
-. Select the option and click *Sync*.
+. To verify the checksum, click *Verify Content Checksum* from the *Action* menu.
+. To perform optimised sync and complete sync, select *Advanced Sync*, perform the following:
+.. Select *Advanced Sync* from the *Select Action* menu.
+.. Select the desired option.
```

Comment:
Do not use sub-steps. There's no reason as the text reads as well and isn't complicated. 
We avoid sub steps if possible. 

IBM Style says never ever use desire in technical writing. 

desired adj
Do not use. Use “required” or “the <item> that you want.”

Is this the point where the user selects from optimised or complete sync? Say that.

Reply:
Hi @melcorr,

Before committing these changes, I had a session with Shweta. Although she wasn't confident about sub-sections, she felt that we are left with no other options and hence, we opted for this. Writing style and word selection are kept the same except for the changes that we took (the last 2 points).

I will replace the word 'desire' with 'required' as I can see the same on IBM Style Guide Quick Reference.

I will also take the changes on the above suggestion where the 'and' can be replaced with 'or'.

At last, I have not taken any decision on the style guide or sentence structure in this commit as it had minor modifications & this can be the probable reason why I am assigned this type of ticket at the current level.

I can understand from your suggestions that there can be a better writing style and sentence structure. But, the advanced sync and verify content checksum are two different actions that are included in the same drop-down menu which can be the probable reason for confusion, as per my understanding.

Please guide me further to make this commit perfect.

Reply:
> Hi @melcorr,
> 
> Before committing these changes, I had a session with Shweta. Although she wasn't confident about sub-sections, she felt that we are left with no other options and hence, we opted for this. Writing style and word selection are kept the same except for the changes that we took (the last 2 points).
I don't understand the "left with no other options" comment because I cannot see the complication that this PR is introducing. 
> 
> I will replace the word 'desire' with 'required' as I can see the same on IBM Style Guide Quick Reference.

If you are selecting between two options, this or that, you could jjust say that. "required" is ok but less descriptive.
My main issue is that desired/required aren't descriptive enough. Do you know what that menu reveals at this point?  
> 
> I will also take the changes on the above suggestion where the 'and' can be replaced with 'or'.
> 
> At last, I have not taken any decision on the style guide or sentence structure in this commit as it had minor modifications & this can be the probable reason why I am assigned this type of ticket at the current level.

That's fair. Everything is a learning opportunity all the same. 
> 
> I can understand from your suggestions that there can be a better writing style and sentence structure. But, the advanced sync and verify content checksum are two different actions that are included in the same drop-down menu which can be the probable reason for confusion, as per my understanding.
> 
> Please guide me further to make this commit perfect.

We've avoided sub-procedures everywhere so far. If all the steps are happening in a sequence, then they can remain in that same sequence. If it is an optional step, which means it mightn't happen at all, it would need to be marked as optional. If the user must choose something, tell the user what the options are. 

I hope this is helping a bit, Malhar.  I understand that you're learning and eager to learn.

Reply:
> If you are selecting between two options, this or that, you could jjust say that. "required" is ok but less descriptive.
> My main issue is that desired/required aren't descriptive enough. Do you know what that menu reveals at this point?

I think the description of all 3 options are there just before the procedure steps. So my question is - the description will still require in the procedure step as well?

> We've avoided sub-procedures everywhere so far. If all the steps are happening in a sequence, then they can remain in that same sequence. If it is an optional step, which means it mightn't happen at all, it would need to be marked as optional. If the user must choose something, tell the user what the options are.
> 
> I hope this is helping a bit, Malhar. I understand that you're learning and eager to learn.

Verify Content Checksum can be the optional step as per my understanding. We can mention it as " (optional) " in procedure steps as per your suggestion. However, I will clear these queries once again with peers.

So, at the end:

1) I will replace require with desire.
2) I will re-verify the options which are mandatory and optional..
3) Sub-steps will be removed and will cover those steps in a single point, even if it requires one more point to be added.

Please let me know if I am missing something.